Release Notes 1.45.5 (#7301)
harness/developer-hub: release-notes/continuous-delivery.md

- While selecting the Deployment type **AWS SAM** and toggling the CV button it was throwing an error due to no template being chosen. The issue is fixed now. (CDS-97771)
- For Winrm deployment with Command step and auth type as Kerberos if the environment variables contains characters `\b, \v, \c, \f, &` the script execution was failing as we parse the response of script execution to xml string and above characters are illegal xml characters. The issue is fixed and now and these characters will be escaped. This change is behind the FF `CDS_ESCAPE_ENV_VARS_FOR_WINRM_KERBEROS_NG`. (CDS-97690, ZD-55276,58201,66326)
- While using a connector in Project scope during pre-flight check, if the check fails connector reference hyperlinks to Connector URL at account level due to which it routes to a Not Found Page. This was happening because scope details were missing from the api. The issue is fixed now. (CDS-97593, ZD-64673)
- Logs were not being uploaded for shell script and some other steps in case the step timed out. The issue is fixed and shell script now uploads the logs on step timeout. (CDS-97521, ZD-64422)

> - Shell script step was not doing a capability check before assigning the tasks to the delegate. The issue is fixed and shell script step will now have host capibility check. (CDS-97512, ZD-66326,66349)

Reviewer: ```suggestion
- The shell script step was not doing a capability check before assigning the tasks to the delegate. The issue is fixed and the shell script step will now have a host capability check. (CDS-97512, ZD-66326,66349)
```

> - ServiceNow step was not updating the ticket yet was showing successful and no logs was being shown. The issue is fixed now and console logs and debugging help has been added in the ServiceNow steps. (CDS-97033, ZD-63637)

Reviewer: ```suggestion
- The ServiceNow step was not updating the ticket but was showing successful with no logs being shown. The issue is now fixed and console logs and debugging help has been added in the ServiceNow steps. (CDS-97033, ZD-63637)
```

</details>

## July 2024

### Version 1.45.5

#### New Features and Enhancement

**Bi-Directional Sync Health Status for Git Sync**

> We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page.md) for more information. (PIPE-16865)

Reviewer: ```suggestion
We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page) for more information. (PIPE-16865)
```

</details>

## July 2024

### Version 1.45.5

> #### New Features and Enhancement

Reviewer 2: ```suggestion
#### New features and enhancements
```

**Bi-Directional Sync Health Status for Git Sync**

We have introduced a Git Sync status page for Harness NG. Once you have synced your Harness entities with your Git repository, you can view bidirectional activity between Harness and your repository using the Bi-Directional Sync Health Status feature. For each entitiy, you can see file path, commit message, status of the last sync as well as the payload. Refer to Harness [Git Sync health Page](/docs/platform/git-experience/git-sync-health-page) for more information. (PIPE-16865)

**HTTP support HTTPS Authentication for Terraform and Terragrunt Modules**

You can now fetch modules for Terraform and Terragrunt via HTTPS authentication using GitHub, Bitbucket, and GitLab connectors(CDS-95361)

> #### Behavior Changes

Reviewer 2: ```suggestion
#### Behavior changes
```

- Previously, when the verify step failed, and an action was taken based on the failure strategy or manual intervention, the title was always shown as **Manual Intervention**. (CDS-97985, ZD-65113)

   Now, the title is updated to reflect the nature of the intervention:
   - **Intervention** is displayed when the action is performed through the CV failure strategy configuration.

   - **Manual Intervention** is displayed when the action is performed through manual intervention.

> #### Fixed Issue

Reviewer 2: ```suggestion
#### Fixed issues
```
